app/main.py

```diff
@@ -2,0 +1,38 @@
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, meaning it is shared among all instances of `Animal`. If the intention is to track all living animals, this is correct. However, if each instance should have its own `alive` list, consider making it an instance variable instead.